### PR TITLE
test

### PR DESCRIPTION
**Pull request by netotz**

### CODE REVIEW

**Review by netotz — COMMENTED** [review]
ok

File: test_.py [hunk 1]
```diff
@@ -7,9 +7,12 @@ def test_add():
 def GetRoberto():
     roberto = 69
 
+def linares_existe():
+    return False
```
Comment (netotz) [inline test_.py:11]: en efecto

File: test_.py [hunk 2]
```diff
@@ -7,9 +7,12 @@ def test_add():
 def GetRoberto():
     roberto = 69
```
Comment (DanielFL0) [inline test_.py:8]: Please refer to the PEP 8 function and variable naming convention.

File: test_.py (hunk 1 above)
Comment (DanielFL0) [inline test_.py:11]: Excelent work!

**Review by DanielFL0 — APPROVED** [review]

**Pull request merged**